content/news/XXXX-XX-XX-zulip-3000_members.md

```diff
@@ -0,0 +1,46 @@
+title: Our Zulip Chat Exceeds 3,000 Members
+authors: Daniel Schürmann
+status: draft
+tags: zulip
+comments: yes
+
+#### Our Zulip Chat Exceeds 3,000 Members
+
+Since moving our community discussions from our mailing list to **Zulip** in 2018, we have never looked back.  
+This month, we have the first time more than 3,000 registered users in our Mixxx Zulip community!
+
+We now exchange around 4,000 messages every month, something a mailing list could never handle effectively. The community continues to grow steadily, both in users and in activity, with over 60 individuals actively participating in discussions every month.  
+
+Zulip has become a solid foundation of our community. A place where developers, translators, designers, and testers meet and collaborate.
+
+### Moving to Zulip
+
+Before Zulip, we used Mailman for our mailing list and a IRC channel for instant messanges. for years, but we ran into several challenges:  
+- Cluttered threading and confusing message flow  
+- No shared cloud storage for files and images  
+- No video chat
+
+As the project and number of contributors grew, we needed a platform that could keep conversations organized and accessible to everyone.
+
+Zulip, which was open-sourced in 2015, was very promising and convinced us after a short evaluation period.
+It's unique thread system was the ideal solution to replace structured email threads and instant chat rooms.
+
+Whenever a topic drifts, we can split it into a new thread, so conversations stay focused and are easy to follow.  
+This approach makes it possible to collaborate asynchronously across time zones without missing important context.
+No need to hang around in chat rooms for not missing an important conversations.
+
+Thanks to Zulip, it’s now much easier to stay up to date. Our team collaboration and community engagement have both
+improved significantly. Zulip works particularly well for open-source projects where many topics run in parallel, and where structure and focus are key to success.
+It can be used with a web frontend of a Mobile app for Android and IOs.
```

Comment:
We should shout out the new mobile app they have release a few month ago, with the full Flutter rewrite IMHO. The new experience is really good!